interface/weno/wenoconnected.php

```diff
@@ -11,15 +11,14 @@
 require_once("../globals.php");
```

Comment:
require_once(__DIR__ . "/../globals.php");
This is almost always best practice. Please change anytime you have opportunity.

Reply:
Ok, I will change these and use them from now on.